tests/e2e/specs/subscriptions/shopper/shopper-subscriptions-manage-payments.spec.js

```diff
@@ -67,39 +54,40 @@ describeif( RUN_SUBSCRIPTIONS_TESTS )(
 			);
 
 			await shopperWCP.goToSubscriptions();
+			await page.click(
+				'.woocommerce-orders-table__cell-subscription-id > a',
```

Comment:
Can we move the selectors to constants?

Reply:
All the tests we have are written in this way. What you've suggested is a good approach by defining selectors early, but I think it would be best if we update it on all flows, not just subscriptions.

Reply:
We can update it in places we are changing right now maybe? Otherwise updating it in all places at once becomes a chore so as a rule, we can just fix this in files we touch moving forward. WDYT?

Reply:
Sure. That works